### ISSUE TITLE
use the banner image used in services in editor as well

### ISSUE THREAD

**Issue by ArnabRaxit** [ess-acppo/nsl-editor#3]
As shown below services uses a banner 
![image](https://user-images.githubusercontent.com/5765791/34862153-f9015288-f7bc-11e7-9cf4-46eb6c8ba512.png)

Put similar banner in editor
![image](https://user-images.githubusercontent.com/5765791/34862170-0a4d32c8-f7bd-11e7-8474-0f6b82b91fd2.png)

**Comment by ess-acppo-djd** [comment]
Not sure if this is the most desirable solution; if we use an image to identify each editor it may need to be resized so that it fits neatly into the area above the editing 'notepad' and to the right of the 'search' button.

Note, also, that this looks after (what we referred to as) 'horizontal branding'.
We also need to be cognisant of the need for 'vertical branding' - identifying between the various environments for each shard (dev, test, UAT, pre-prod, Prod).

It appears the 'Dev editor' tag in the top left corner of the editor may already have catered for distinguishing between 'vertical' environments, but this is also something that needs to be ascertained.

One additional item that has been discussed with respect to the branding and white-labelling issues, is the colour scheme for each shard. A preferred option is to provide a 'base colour' and have the style sheets generate the palette for all pages and windows associated with the shard.

**Comment by ArnabRaxit** [comment]
> It appears the 'Dev editor' tag in the top left corner of the editor may already have catered for distinguishing between 'vertical' environments, but this is also something that needs to be ascertained.

That's right. The 'Dev editor' tag will read 'NXL editor' in prod environment. Also, the color will change based on the vertical environment.

**Comment by ArnabRaxit** [comment]
The colors of the vertical schemes are currently fixed at the following:
development-color: [#696076](https://www.google.com.au/search?q=%23696076); 
test-color: [#663399](https://www.google.com.au/search?q=%23663399);
staging-color: [#6699cc](https://www.google.com.au/search?q=%236699cc);

@ess-acppo-djd : Do you think making these configurable is a good idea?
